### ISSUE TITLE
more python upgrade trouble: pycolt and theodore

### ISSUE THREAD

**Issue by markuskowa** [Nix-QChem/NixOS-QChem#540]
Fails now in the evaluation phase with:
```
error: pytest-runner has been removed as it uses deprecated features of setuptools and is deprecated by upstream
```

CC @sheepforce

**Comment by sheepforce** [comment]
Given the package is somehow build in a standard way (a big "if", I know), [the correct solution is the `pythonRelaxDepsHook`](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook) and the `removeDeps` list. For `setup.py` this should work.

**Comment by sheepforce** [comment]
Closed via #541
